armbian/base/scripts/systemd-startup-checks.sh

```diff
@@ -80,3 +80,9 @@ mkdir -p /mnt/ssd/system/journal/
 # set permissions for whole ssd 
 # (user:rwx group:r-x other:---)
 chmod -R 750 /mnt/ssd
+
```

Comment:
Are these diffs in `armbian/base/scripts/` intended for this PR? Maybe it'd be better to send these changes as separate PRs, to not mix the significant documentation changes with some smaller technical changes?

Reply:
Good point, will move them.